pkg/gce-pd-csi-driver/node.go

```diff
@@ -741,6 +748,15 @@ func (ns *GCENodeServer) GetVolumeLimits() (int64, error) {
 			return volumeLimitSmall, nil
 		}
 	}
+
+	// Get attach limit override from label
+	attachLimitOverride, err := GetAttachLimitsOverrideFromNodeLabel(ctx, ns.MetadataService.GetName())
+	if err == nil && attachLimitOverride > 0 && attachLimitOverride < 128 {
+		return attachLimitOverride, nil
+	}
+	// If there is an error or the range is not valid, still proceed to get defaults for the machine type
```

Comment:
Should at least log a warning so a user will have a clue why their CSINode attach limit isn't what they expect.